src/react-apps/applications/shared/src/components/AltinnFilterChip.tsx

```diff
@@ -0,0 +1,90 @@
+import { Chip, createMuiTheme } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import altinnTheme from '../theme/altinnStudioTheme';
+import * as React from 'react';
+
+export interface IAltinnFilterChipCompontentProvidedProps {
+  classes: any;
+  key: any;
+  label: string;
+  onclickFunction?: any;
+  active: boolean;
+  className?: string;
+  deleteIcon?: any;
+  onDeleteFunction?: any;
+  sortIcon?: boolean;
+}
+
+export interface IAltinnFilterChipComponentProps extends IAltinnFilterChipCompontentProvidedProps {
```

Comment:
Unecessary interface, since it doesn't expand anything.

Reply:
removed